shell.nix

```diff
@@ -47,6 +48,7 @@ lib.overrideDerivation postgrest.env (
         complete -F _command postgrest-with-postgresql-10
         complete -F _command postgrest-with-postgresql-9.6
         complete -F _command postgrest-with-postgresql-9.5
+        source ${postgrest.hsie.bashCompletion}
```

Comment:
This looks like I should be getting bash completion for `hsie`. But it doesn't work for me. I can only complete filenames, but not commands etc. - is this supposed to work?

Reply:
I played around with the completion script's content a bit and I get this:
```bash
$ hsie --bash-completion-index dump-imports
option --bash-completion-index: cannot parse value `dump-imports'
```

Not sure whether that's the right way to call the completion-index subcommand - but the error looks like it might be related to the error I get for `check-wildcards`?

Reply:
Ah, this is because it needs a number, not a string. `--bash-completion-index 0` etc. works.

Reply:
Should work with the latest commit, now that it completes for the right executable name... :-)